binaries not found after make install

From build #7 onward (a16d224594f5dbe115dc7e0195fd87240120a3fd), the compilation works, but functional tests are failing.

basically, the script can't find the binaries.

The output is as follows : 

```
make[4]: Entering directory `/var/lib/jenkins/workspace/torque/ARCH/x86_64/OS/u1404/SITE/generic/VERSION/2.5.13/torque-2.5.13/src/server'
/bin/bash ../../buildutils/pbs_mkdirs aux || :
mkdir: cannot create directory ‘/var/spool/torque’: Permission denied
/bin/bash ../../buildutils/pbs_mkdirs server
mkdir: cannot create directory ‘/var/spool/torque’: Permission denied
make[4]: *** [install-exec-hook] Error 1
make[4]: Leaving directory `/var/lib/jenkins/workspace/torque/ARCH/x86_64/OS/u1404/SITE/generic/VERSION/2.5.13/torque-2.5.13/src/server'
make[3]: *** [install-exec-am] Error 2
make[3]: Leaving directory `/var/lib/jenkins/workspace/torque/ARCH/x86_64/OS/u1404/SITE/generic/VERSION/2.5.13/torque-2.5.13/src/server'
make[2]: *** [install-am] Error 2
make[2]: Leaving directory `/var/lib/jenkins/workspace/torque/ARCH/x86_64/OS/u1404/SITE/generic/VERSION/2.5.13/torque-2.5.13/src/server'
make[1]: *** [install-recursive] Error 1
make[1]: Leaving directory `/var/lib/jenkins/workspace/torque/ARCH/x86_64/OS/u1404/SITE/generic/VERSION/2.5.13/torque-2.5.13/src'
make: *** [install-recursive] Error 1
```
seems like some config has to be changed in `build.sh`

In the config.log, as a hint, we have : 

```
make[4]: Entering directory `/var/lib/jenkins/workspace/torque/ARCH/x86_64/OS/u1404/SITE/generic/VERSION/2.5.13/torque-2.5.13/src/server'
/bin/bash ../../buildutils/pbs_mkdirs aux || :
mkdir: cannot create directory ‘/var/spool/torque’: Permission denied
/bin/bash ../../buildutils/pbs_mkdirs server
mkdir: cannot create directory ‘/var/spool/torque’: Permission denied
make[4]: *** [install-exec-hook] Error 1
make[4]: Leaving directory `/var/lib/jenkins/workspace/torque/ARCH/x86_64/OS/u1404/SITE/generic/VERSION/2.5.13/torque-2.5.13/src/server'
make[3]: *** [install-exec-am] Error 2
make[3]: Leaving directory `/var/lib/jenkins/workspace/torque/ARCH/x86_64/OS/u1404/SITE/generic/VERSION/2.5.13/torque-2.5.13/src/server'
make[2]: *** [install-am] Error 2
make[2]: Leaving directory `/var/lib/jenkins/workspace/torque/ARCH/x86_64/OS/u1404/SITE/generic/VERSION/2.5.13/torque-2.5.13/src/server'
make[1]: *** [install-recursive] Error 1
make[1]: Leaving directory `/var/lib/jenkins/workspace/torque/ARCH/x86_64/OS/u1404/SITE/generic/VERSION/2.5.13/torque-2.5.13/src'
make: *** [install-recursive] Error 1
```

--- a/build.sh
+++ b/build.sh
@@ -45,6 +45,6 @@ cd $WORKSPACE/$NAME-$VERSION
 echo "cleaning up previous builds"
 #make distclean
 echo "Configuring the build"
-CC=`which gcc` CXX=`which g++` ./configure --prefix=${SOFT_DIR} --without-tcl 
+CC=`which gcc` CXX=`which g++` ./configure --prefix=${SOFT_DIR} --without-tcl --with-server-home=${SOFT_DIR}/spool
 echo "Running the build"
 make all

